src/test/java/core/basesyntax/operation/impl/AdditionHandlerImplTest.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.operation.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.AdditionHandlerImpl;
+import core.basesyntax.service.operation.Handler;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AdditionHandlerImplTest {
+    private static Handler addHandler;
+
+    @BeforeClass
+    public static void setUp() {
+        addHandler = new AdditionHandlerImpl();
+    }
+
+    @Test
+    public void operate_validDataReturnAndSupply_ok() {
```

Comment:
**Check and fix ALL method names.** Use pattern `testedMethodName_description_ok/notOk`
Description is not obligate for OK
```suggestion
    public void handle_validData_ok() {
```

Reply:
Rommelua,thaks .Now  I understood.